unity-renderer/Assets/UnityGLTF/Scripts/GLTFThrottlingCounter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using DCL;
+using UnityEngine;
+
+namespace UnityGLTF
+{
+    public class GLTFThrottlingCounter
+    {
+        public static GLTFThrottlingCounter i => new GLTFThrottlingCounter();
```

Comment:
Does this mean that each time we access to `GLTFThrottlingCounter.i` a new instance of `GLTFThrottlingCounter` is being created? If it is that we want, why we need to manage an `i` static instance of `GLTFThrottlingCounter`? Couldn't we simply create the instance of the class directly from the consumer that need it?

Reply:
I agree! Since we are moving out from singletons this seems a little out of the common pattern

Reply:
I think you guys solved me a bug. 🙏 

`GLTFThrottlingCounter` is supposed to be a singleton.

Regarding the choice of having to implement a singleton when we are trying to get rid of them is a bit nuanced. 
The thing is we can't make the GLTF loader know about `Main` `Environment` etc. Because its supposed to be a completely self contained piece of software. But at the same time, the loader needs persistent data, so for these use cases we end up again having to rely on static classes, singletons, etc.

At the same time, think that the GLTF loader doesn't have a "global" entry point. To use it we just create the `GLTFComponent` and its good to go. So, again we end up relying on singletons for having to share state between loaders--and that is what the `GLTFThrottlingCounter` does.

I accept suggestions on the matter, however I don't think a solution is possible that is on the scope on this PR, we can discuss this async.

Reply:
Managed to address this in the end. Resolving.